code/_onclick/hud/maptext.dm

```diff
@@ -0,0 +1,29 @@
+/obj/screen/maptext
+	screen_loc = "NORTH:14,WEST"
+	plane = HUD_PLANE
+	layer = HUD_BASE_LAYER
+	maptext_height = 32
+	maptext_width = 480
+	var/last_word = null
+
+/client/MouseEntered(object, location)
```

Comment:
документация такая

> This is called when no mouse buttons are pressed while pointing to the object. 

Don't define this unless you need it, because it generates extra communication that is otherwise avoided. Defining it on only the objects that require it reduces overhead. 

разработчик такой

![image](https://github.com/ChaoticOnyx/OnyxBay/assets/45202681/c33e780d-5ca6-4c40-add7-3df1cd540003)